includes/Helpers/BanHelper.php

```diff
@@ -173,18 +173,17 @@ private function getBansForRequestFromDatabase(Request $request): array
 SELECT b.* FROM ban b
 LEFT JOIN netmask n ON 1 = 1
     AND n.cidr = b.ipmask
-    AND n.protocol = CASE LENGTH(b.ip) WHEN 4 THEN 4 WHEN 16 THEN 6 END
+    AND n.protocol = CASE WHEN INET6_ATON(b.ip) IS NOT NULL THEN 6 ELSE 4 END
```

Comment:
It looks like [`inet6_aton`][1] returns a non-null value for IPv4 addresses too?


[1]: https://mariadb.com/kb/en/inet6_aton/